webpack.config.js

```diff
@@ -53,12 +53,43 @@ const webpackConfig = {
 					fallback: 'style-loader',
 					use: [
 						'css-loader',
+						{
+							loader: 'postcss-loader',
+							options: {
+								plugins: [
+									require( './node_modules/@wordpress/postcss-themes' )( {
```

Comment:
I merged a change that adds `autoprefixer` and `postcss-loader`. These changes can actually go in [`postcss.config.js`](https://github.com/woocommerce/wc-admin/blob/04e6ff6cdf8cb99879d98c7f9d82d4f8afb3198f/postcss.config.js) now instead.